packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -281,6 +284,12 @@ mixin RendererBinding on BindingBase, ServicesBinding, SchedulerBinding, Gesture
     }
   }
 
+  void _handleWebFirstFrame(Duration _) {
+    assert(kIsWeb);
+    const MethodChannel methodChannel = MethodChannel('flutter/service_worker');
```

Comment:
This looks like a [system_channel](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/system_channels.dart), but I guess it's fine here since it's used only once at this time.

Reply:
yeah, I think since this is sort of private it would be good to keep the scope limited